impeller/display_list/dl_dispatcher.cc

```diff
@@ -1074,7 +1074,7 @@ void DlDispatcher::drawDisplayList(
     canvas_.SaveLayer(save_paint);
   }
 
-  if (display_list->has_rtree()) {
+  if (display_list->has_rtree() && !initial_matrix_.HasPerspective()) {
```

Comment:
Nit: Add todo comment nearby linking to the new bug?

Reply:
Done